www_src/pages/editor/editor.jsx

```diff
@@ -48,5 +48,96 @@ var Editor = React.createClass({
   }
 });
 
+var TextEditor = React.createClass({
+  mixins: [React.addons.LinkedStateMixin, Binding],
+  getInitialState: function () {
+    return {
+      fontSize: 18,
+      fontFamily: 'Roboto',
+      color: '#645839',
+      fontWeight: 'normal',
+      fontStyle: 'normal',
+      textDecoration: 'none',
+      textAlign: 'center'
+    };
+  },
+  onChange: function (id) {
+    return (e) => {
+      var state = {};
```

Comment:
If we're clearing and then setting a single property, we could do that directly as a `var state = { id: e.target.checked };`, although state does not "clear" with setState, it simply updates any keys provided, so in this case it'd just be `this.setState({ id: e.target.checked });`